Documentation/check-cmdref.sh

```diff
@@ -9,6 +9,6 @@ cmdref_dir="${script_dir}/cmdref"
 
 if ! git diff --quiet -- "${cmdref_dir}" ; then
     git --no-pager diff "${cmdref_dir}"
-    echo "HINT: to fix this, run 'make -C Documentation update-cmdref ; git commit Documentation/cmdref --message \"docs: Update cmdref\"'"
+    echo "HINT: to fix this, run 'make -C Documentation update-cmdref ; git commit Documentation/cmdref -s --message \"docs: Update cmdref\"'"
```

Comment:
Should we just remove the `git commit` part? I'd expect this to be committed with the actual source changes, not as a separate commit.